packages/cli/scripts/generate-test-tables.ts

```diff
@@ -16,10 +15,10 @@ const config = defineStore({
     Enum2: ["E1"],
   },
   userTypes: {
-    TestTypeAddress: { filePath: "./contracts/src/types.sol", type: "address" },
-    TestTypeInt64: { filePath: "./contracts/src/types.sol", type: "int64" },
-    TestTypeBool: { filePath: "./contracts/src/types.sol", type: "bool" },
-    TestTypeUint128: { filePath: "./contracts/src/types.sol", type: "uint128" },
+    TestTypeAddress: { filePath: "../contracts/src/types.sol", type: "address" },
```

Comment:
previously, codegen expected this path to be relative to `process.cwd()` (almost always the project root/same dir as the `mud.config.ts`)

once I changed the paths to be relative to the project root/`mud.config.ts`, this broke, because for this particular script, the config is in a slightly different place